### PR TITLE
Add feature routing

### PR DESCRIPTION
**Pull request by beatfreaker**
added routing feature
- Used react-router and react-router-dom
- Created sample About page
- Added link of About page in Home page so that user can navigate to About page

### CODE REVIEW

**Comment by beatfreaker** [comment]
Added references in [README.md](https://github.com/beatfreaker/hacktober-learn-react/blob/d1ae2af615515caf6af68fe556a39e6f0041d5ee/README.md#references)
like this we can list the resources which we referred while developing this app.

**Comment by huiyie** [comment]
@TheCodeArtist What is the best way to resolve the conflicts with this PR? Should I rebase based on @beatfreaker 's commits?

**Comment by TheCodeArtist** [comment]
Hmmm... I see that PR #13 has been merged into `master`.
So, there are a few conflicts in this PR #14 now.

Fortunately there is no logical conflict.

PR#13 
* Updates the header
* Updates the content of the Home page

PR#14
* Add routing links
* Adds a Home Page
* Adds an About page.

Usually the author of the commits is in the best position to update/rebase the PR to match the codebase.
@beatfreaker Do you want to try updating the commtis in this PR to latest `huiyie/hacktober-learn-react:master` ?

If not then, either @huiyie or i can git fetch origin pull/14/head:master and work on rebasing the changes in local tree and the push updated PR / another PR (depending on whether github permits us to).

**Comment by beatfreaker** [comment]
@huiyie you can manually resolve conflicts
```js
<<<<<<< feature-routing
        <Main />
=======
        <p className="App-intro">
          To get started, open <code>CONTRIBUTING.md</code>.
        </p>
>>>>>>> master
```
replace above with below code
```js
        <Main />
```

**Comment by beatfreaker** [comment]
@huiyie @TheCodeArtist update the PR, please review it.

File: src/App.css [hunk 1]
```diff
@@ -2,30 +2,14 @@
...
-  padding: 0px;
-}
-
 .App-logo {
-  animation: App-logo-spin infinite 30s linear;
-  position: absolute;
-  left: -40px;
-  top: -350px;
-  height: 280px;
+  animation: App-logo-spin infinite 20s linear;
```
Comment (TheCodeArtist) [inline src/App.css]: @beatfreaker Can these changes be retained. They are used by the Header elements in `App.js`

File: src/App.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ class App extends Component {
           <div className="Logo-header">
             <img src={headerBackground} className="App-logo-back" alt="logo-back" />
             <img src={logo} className="App-logo" alt="logo" />
-	  </div>
+	        </div>
```
Comment (TheCodeArtist) [inline src/App.js]: Fix this unaligned extra white-space?...

Comment (beatfreaker) [reply src/App.js]: @TheCodeArtist for to push to changes .... done in latest commit.

**Pull request merged**